packages/story-editor/src/output/utils/HiddenAudio.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { ResourcePropTypes } from '@googleforcreators/media';
+
+/**
+ * Internal dependencies
+ */
+import { BackgroundAudioPropType } from '../../types';
+
+function HiddenAudio({ backgroundAudio, tracks, id }) {
+  const { mimeType, src } = backgroundAudio;
+
+  const sourceProps = {
+    type: mimeType,
+    src,
+  };
+
+  const videoProps = {
+    autoPlay: 'autoplay',
+    layout: 'responsive',
+    width: '0',
+    height: '0',
+    loop: 'loop',
+  };
```

Comment:
Since these are static we could just add them directly inline to `<amp-video` below

Reply:
Also, looks like we can just provide `layout="nodisplay"`. No width or height necessary

Reply:
> Also, looks like we can just provide layout="nodisplay". No width or height necessary

This based on your recommendations.

Reply:
I know. Because I first thought `layout="nodisplay"` does not work, but it does appear to work. Now this is my new recommendation.